src/sessions.py

```diff
@@ -0,0 +1,59 @@
+import json
+import requests
+
+def list_sessions(bdb):
+    """Lists all saved sessions with the number of entries in each, and whether
+    they were sent or not."""
+    return bdb.execute('select * from bayesdb_session;')
+
+
+def current_session_id(bdb):
+    """Returns the current integer session id."""
+    return bdb.session_id
+
+
+def dump_session_as_json(bdb, session_id):
+    """Returns a JSON string representing the list of SQL or BQL entries (e.g.
+    queries) executed within session `session_id`."""
+    current_session = current_session_id(bdb)
+    if session_id > current_session or session_id < 1:
+        raise ValueError('No such session (%d)' % session_id)
+    bql = """SELECT * FROM bayesdb_session_entries
+    WHERE session_id == ?
+    ORDER BY time DESC"""
+    entries = bdb.sql_execute(bql, (session_id,))
```

Comment:
entries is perhaps better named cursor, to make the following line a bit clearer?